front/src/reducers/osrdconf/selectors.ts

```diff
@@ -62,7 +62,7 @@ export const getGridMarginBefore = (state: RootState) => getSection(state).gridM
 export const getGridMarginAfter = (state: RootState) => getSection(state).gridMarginAfter;
 export const getStandardStdcmAllowance = (state: RootState) =>
   state.osrdconf.stdcmConf.standardStdcmAllowance;
-export const getPowerRestriction = (state: RootState) => getSection(state).powerRestriction;
+export const getPowerRestriction = (state: RootState) => getSection(state).powerRestrictionRanges;
```

Comment:
Why not using `getPowerRestrictionRanges` ?

Reply:
Done